readme.md

```diff
@@ -88,6 +89,35 @@ TODO: add application schema
 
 3. packages - separate app features or functionalities
 
+### 5.2 Mobile
+
```

Comment:
Add mention here that we build only for android like: This project is mainly focused on the Android platform.